spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/converter/PubSubMessageConverter.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.pubsub.support.converter;
+
+import java.io.IOException;
+
+import com.google.pubsub.v1.PubsubMessage;
+
+/**
+ * Interface for converters that can convert POJOs to and from PubSub messages.
+ *
+ * @author Chengyuan Zhao
+ */
+public interface PubSubMessageConverter {
+
+	/**
+	 * Convert a Java object to a byte array payload.
+	 * @param object the object to convert
+	 * @return a byte array ready to be sent with PubSub
+	 */
+	byte[] toPayload(Object object) throws IOException;
```

Comment:
I think this one has to be aligned with other `MessageConverter`s. Also for the simetry this one  has to deal with the `PubsubMessage`:

```
PubsubMessage toMessage(Object payload, Map headers);
```

Reply:
done